guide/7.texi

```diff
@@ -663,6 +663,7 @@ As part of the @code{TIMES} clause of a @statementref{PERFORM}.
 @comment *********************************************************************
 @page
 @newsection{7.8,GnuCOBOL Statements}
+@iftex
```

Comment:
why should this be removed for non-tex - and what is that?

Reply:
It makes `makeinfo --html` segfault, and I have no idea why... Given that these lines are not useful in html mode, it was the simplest solution...

Reply:
please add a `@comment` before that with an explanation and ideally reference the upstream bug report, too

Reply:
done